gematria/datasets/exegesis_benchmark_lib_test.cc

```diff
@@ -385,14 +384,18 @@ TEST_F(ExegesisBenchmarkTest, TestBenchmarkAdd) {
 }
 
 TEST_F(ExegesisBenchmarkTest, TestProcessAnnotatedBlock) {
-  BlockAnnotations Annotations = {
-      .code_location = 0xff,
-      .block_size = 4096,
-      .block_contents = 0xff,
-      .accessed_blocks = {0xaa, 0xbb},
-      .initial_regs = {{.register_index = 0, .register_value = 5},
-                       {.register_index = 1, .register_value = 17}},
-      .loop_register = std::nullopt};
+  ExecutionAnnotations Annotations;
+  Annotations.set_code_location(0xff);
+  Annotations.set_block_size(4096);
+  Annotations.set_block_contents(0xff);
+  Annotations.add_accessed_blocks(0xaa);
+  Annotations.add_accessed_blocks(0xbb);
+  RegisterAndValue *register_a = Annotations.add_initial_registers();
+  register_a->set_register_index(0);
+  register_a->set_register_value(5);
+  RegisterAndValue *register_b = Annotations.add_initial_registers();
+  register_b->set_register_index(1);
+  register_b->set_register_value(17);
```

Comment:
Optional: More compact way to do this is with [`ParseProto()`](https://github.com/google/gematria/blob/129049615bee55ea38f5ebf574b0529bffd33815/gematria/testing/parse_proto.h#L64):
```cc
const ExecutionAnnotations Annotations = ParseTextProto(
    R"pb(
      code_start_addrsss: 0xff
      block_size: 4096
      block_contents: 0xff
      # ...
      initial_registers {
        register_name: "RAX"
        register_value: 17
      }
    )pb"
);
```

Same below.